lib/phobos/producer.rb

```diff
@@ -67,18 +67,33 @@ def kafka_client
           producer_store[:kafka_client]
         end
 
+        def create_sync_producer
```

Comment:
https://github.com/phobos/phobos/diffs/1?base_sha=fcf14a6954061a01b63b22173dc58a5814f134be&commentable=true&pull_number=112&sha1=fcf14a6954061a01b63b22173dc58a5814f134be&sha2=97cac39158eec5dc0ffb7fec78e605e099278c9c&short_path=04c6e90&unchanged=expanded&utf8=%E2%9C%93#usage-producing-messages-to-kafka

> Regular producers will deliver the messages synchronously and disconnect, it doesn't matter if you use publish or publish_list after the messages get delivered the producer will disconnect.
> 
> Async producers will accept your messages without blocking, use the methods async_publish and async_publish_list to use async producers.

The change you have made kind of contradicts the description of the "regular" Producer as described by the Readme.

If you simply change from using "regular" producer to async producer, don't you achieve the exactly the same thing and reap the 30% speed increase without sacrificing code simplicity?